sdk/communication/AzureCommunicationCalling/AzureCommunicationCalling-Swift.h

```diff
@@ -0,0 +1,1027 @@
+
+    optional func remoteParticipant(_ remoteParticipant: RemoteParticipant, didUpdateVideoStreams args: RemoteVideoStreamsEventArgs)
+}
+
+
+/**
+ * A set of methods that are called by ACSIncomingCall in response to important events.
+ */
+public protocol IncomingCallDelegate : NSObjectProtocol {
+
+    
+    optional func incomingCall(_ incomingCall: IncomingCall, didEndCall args: PropertyChangedEventArgs)
+}
+
+
+/**
+ * A set of methods that are called by ACSDeviceManager in response to important events.
+ */
+public protocol DeviceManagerDelegate : NSObjectProtocol {
+
+    
+    optional func deviceManager(_ deviceManager: DeviceManager, didUpdateCameras args: VideoDevicesUpdatedEventArgs)
+}
+
+
+/// Property bag class for Video Options. Use this class to set video options required during a call (start/accept/join)
+open class VideoOptions : NSObject {
+
+    public init(localVideoStream: LocalVideoStream)
+
+    
+    /// Deallocates the memory occupied by this object.
+    open func dealloc()
+
+    
+    /// The video stream that is used to render the video on the UI surface
+    open var localVideoStream: LocalVideoStream
+}
+
+
+/// Local video stream information
+open class LocalVideoStream : NSObject {
+
+    public init(camera: VideoDeviceInfo)
+
+    
+    /// Deallocates the memory occupied by this object.
+    open func dealloc()
+
+    
+    /// Video device to use as source for local video.
+    open var source: VideoDeviceInfo { get }
+
+    
+    /// Sets to True when the local video stream is being sent on a call.
+    open var isSending: Bool { get }
+
+    
+    /// MediaStream type being used for the current local video stream (Video or ScreenShare).
+    open var mediaStreamType: MediaStreamType { get }
+
+    
+    // Class extension begins for LocalVideoStream.
+    open func switchSource(camera: VideoDeviceInfo, completionHandler: @escaping (Error?) -> Void)
+}
+
+// Class extension ends for LocalVideoStream.
+
+
+/// Information about a video device
+open class VideoDeviceInfo : NSObject {
+
+    
+    /// Deallocates the memory occupied by this object.
+    open func dealloc()
+
+    
+    /// Get the name of this video device.
+    open var name: String { get }
+
+    
+    /// Get Name of this audio device.
+    open var id: String { get }
+
+    
+    /// Direction of the camera
+    open var cameraFacing: CameraFacing { get }
+
+    
+    /// Get the Device Type of this video device.
+    open var deviceType: VideoDeviceType { get }
+}
+
+
+/// Property bag class for Audio Options. Use this class to set audio settings required during a call (start/join)
+open class AudioOptions : NSObject {
+
+    public init()
+
+    
+    /// Deallocates the memory occupied by this object.
+    open func dealloc()
+
+    
+    /// Start an outgoing or accept incoming call muted (true) or un-muted(false)
+    open var muted: Bool
```

Comment:
Should probably be `isMuted` to align with other boolean properties in your SDK?  Perhaps `startMuted`?